src/Pages/URLShortener/URLShortener.js

```diff
@@ -344,6 +348,20 @@ export default function URLShortenerPage(props) {
   // the below input layout is taken from
   // https://tailwindui.com/components/application-ui/forms/form-layouts
     <div className='overview-container bg-gradient-to-r from-gray-800 to-gray-600 min-h-[100dvh]'>
+      <ConfirmationModal {... {
+        headerText: `Delete ${urlToDelete.alias} for ${urlToDelete.url} ?`,
+        bodyText: `Are you sure you want to delete 
+          ${urlToDelete.alias}? It'll be gone forever if you do.`,
+        confirmText: `Yes, goodbye ${urlToDelete.alias}`,
+        cancelText: 'No, it\'s chill',
+        confirmClassAddons: 'bg-red-600 hover:bg-red-500',
+        handleConfirmation: () => {
```

Comment:
what about when they cancel? does the modal close?

Reply:
done